x-pack/test/cases_api_integration/security_and_spaces/tests/common/kibana_alerting_cases_index.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  MAIN_SAVED_OBJECT_INDEX,
+  ALERTING_CASES_SAVED_OBJECT_INDEX,
+} from '@kbn/core-saved-objects-server';
+import {
+  CASE_COMMENT_SAVED_OBJECT,
+  CASE_CONFIGURE_SAVED_OBJECT,
+  CASE_SAVED_OBJECT,
+  CASE_USER_ACTION_SAVED_OBJECT,
+} from '@kbn/cases-plugin/common/constants';
+import {
+  deleteAllCaseItems,
+  getCaseCommentSavedObjectsFromES,
+  getCaseSavedObjectsFromES,
+  getCaseUserActionsSavedObjectsFromES,
+  getConfigureSavedObjectsFromES,
+} from '../../../common/lib/api';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const esArchiver = getService('esArchiver');
+  const es = getService('es');
+
+  describe('Kibana index: Alerting & Cases', () => {
+    before(async () => {
+      await esArchiver.load('x-pack/test/functional/es_archives/cases/migrations/8.8.0');
```

Comment:
Would it be possible to use kbn archiver here instead?

Reply:
Yeah, for migrations, we're forced to use esArchiver to load the old index state unfortunately

(FWIW this PR has been merged against the feature branch - https://github.com/elastic/kibana/pull/154888)